rotkehlchen/tests/unit/decoders/test_gmx.py

```diff
@@ -0,0 +1,187 @@
+            location_label=user_address,
+            notes=f'Burned {gas_amount} ETH for gas',
+            counterparty=CPT_GAS,
+        ), EvmEvent(
+            tx_hash=evmhash,
+            sequence_index=1,
+            timestamp=timestamp,
+            location=Location.ARBITRUM_ONE,
+            event_type=HistoryEventType.DEPOSIT,
+            event_subtype=HistoryEventSubType.DEPOSIT_ASSET,
+            asset=A_ETH,
+            balance=Balance(FVal(amount_deposited)),
+            location_label=user_address,
+            notes=f'Increase long position with {amount_deposited} ETH in GMX',
+            address=GMX_POSITION_ROUTER,
+            counterparty=CPT_GMX,
+        ), EvmEvent(
+            tx_hash=evmhash,
+            sequence_index=3,
+            timestamp=timestamp,
+            location=Location.ARBITRUM_ONE,
+            event_type=HistoryEventType.SPEND,
+            event_subtype=HistoryEventSubType.FEE,
+            asset=A_ETH,
+            balance=Balance(FVal(fee_amount)),
+            location_label=user_address,
+            notes=f'Spend {fee_amount} ETH as GMX fee',
+            counterparty=CPT_GMX,
+            address=GMX_POSITION_ROUTER,
+        ),
+    ]
+
+
+@pytest.mark.vcr()
+@pytest.mark.parametrize('arbitrum_one_accounts', [['0x3D4d8A52D5717b09CA1e1980393d244Ac258C6AA']])
+def test_decrease_short_in_gmx(database, arbitrum_one_inquirer, arbitrum_one_accounts):
+    evmhash = deserialize_evm_tx_hash('0xa3dd437d57689479db67ce685c4b70ee9bdbfbcc53f99fc343f73c89edaafe7a')  # noqa: E501
+    events, _ = get_decoded_events_of_transaction(
+        evm_inquirer=arbitrum_one_inquirer,
+        database=database,
+        tx_hash=evmhash,
+    )
+    user_address = arbitrum_one_accounts[0]
+    amount_withdrawn, fee_amount = '140.09783', '0.000215'
+    timestamp = TimestampMS(1705498390000)
+    a_usdc = Asset('eip155:42161/erc20:0xaf88d065e77c8cC2239327C5EDb3A432268e5831')
+    assert events == [
+        EvmEvent(
+            tx_hash=evmhash,
+            sequence_index=18,
+            timestamp=timestamp,
+            location=Location.ARBITRUM_ONE,
+            event_type=HistoryEventType.WITHDRAWAL,
+            event_subtype=HistoryEventSubType.REMOVE_ASSET,
+            asset=a_usdc,
+            balance=Balance(FVal(amount_withdrawn)),
+            location_label=user_address,
+            notes=f'Decrease short position withdrawing {amount_withdrawn} USDC in GMX',
+            address=GMX_POSITION_ROUTER,
+            counterparty=CPT_GMX,
+        ), EvmEvent(
+            tx_hash=evmhash,
+            sequence_index=20,
+            timestamp=timestamp,
+            location=Location.ARBITRUM_ONE,
+            event_type=HistoryEventType.SPEND,
+            event_subtype=HistoryEventSubType.FEE,
+            asset=A_ETH,
+            balance=Balance(FVal(fee_amount)),
+            location_label=user_address,
+            notes=f'Spend {fee_amount} ETH as GMX fee',
```

Comment:
fee is always in ETH?

Reply:
No, it depends on the pair queried. This was for longing ETH and the fee is in ETH